container/new_push.bzl

```diff
@@ -169,13 +145,12 @@ def _impl(ctx):
 new_container_push = rule(
     attrs = dicts.add({
         "format": attr.string(
-            default = "legacy",
+            default = "Docker",
             values = [
-                "oci",
-                "docker",
-                "legacy",
+                "OCI",
```

Comment:
are we losing any functionality by not supporting legacy anymore? If so, should we document this better, and even possibly cut a release before this.

Reply:
No we are not losing any functionality. This change just treats the attributes differently. What was supported by `legacy` before is now supported by `Docker`. This makes `new_container_push` a drop-in replacement for `container_push`. Since `new_container_push` is experimental, I don't see a reason to cut a release because the API is subject to change and we didn't *remove* any functionality.